azafea/event_processors/endless/metrics/tests_v3/integration/test_metrics_v3.py

```diff
@@ -59,7 +67,21 @@ def test_request(self):
         # Ensure the record was inserted into the DB
         with self.db as dbsession:
             channel = dbsession.query(Channel).one()
-            assert channel.image_id == 'image_id'
+            assert channel.image_id == 'eos-eos3.7-amd64-amd64.190419-225606.base'
+            assert channel.image_product == 'eos'
+            assert channel.image_arch == 'amd64'
+            assert channel.image_branch == 'eos3.7'
+            assert channel.image_platform == 'amd64'
+            assert channel.image_timestamp == datetime(
+                2019, 4, 20, 0, 56, 6, tzinfo=timezone(timedelta(seconds=7200))
```

Comment:
You can use `hours` instead of `seconds` (but the whole timezone may actually be removed, see below).